components/gc-ministers/index.json-ld

```diff
@@ -0,0 +1,169 @@
+					}
+				],
+				"fr": [
+					{
+						"@type": "source-code",
+						"description": "Exemple de code :",
+						"code": "<section class=\"ministers\">\n\t<h2>Ministre(s)</h2>\n\t<div class=\"row wb-eqht-grd\">\n\t\t<div class=\"col-md-6\">\n\t\t\t<div class=\"row\">\n\t\t\t\t<div class=\"pull-right col-md-7 col-xs-12\">\n\t\t\t\t\t<h3><a href=\"#\">L'honorable [prénom et nom de famille]</a></h3>\n\t\t\t\t</div>\n\t\t\t\t<div class=\"col-md-5 col-xs-12\">\n\t\t\t\t\t<img src=\"./img/minister-200x250.jpg\" alt=\"L'honorable [prénom et nom de famille]\">\n\t\t\t\t</div>\n\t\t\t\t<div class=\"col-md-7 col-xs-12\">\n\t\t\t\t\t<p>[Titre officiel du ministre]</p>\n\t\t\t\t\t<ul>\n\t\t\t\t\t\t<li><a href=\"\">Lettre de mandat [facultatif]</a></li>\n\t\t\t\t\t\t<li><a href=\"\">Dossier d'information [facultatif]</a></li>\n\t\t\t\t\t</ul>\n\t\t\t\t</div>\n\t\t\t</div>\n\t\t</div>\n\t\t<div class=\"col-md-6\">\n\t\t\t<div class=\"row\">\n\t\t\t\t<div class=\"pull-right col-md-7 col-xs-12\">\n\t\t\t\t\t<h3><a href=\"#\">L'honorable [prénom et nom de famille]</a></h3>\n\t\t\t\t</div>\n\t\t\t\t<div class=\"col-md-5 col-xs-12\">\n\t\t\t\t\t<img src=\"./img/minister-200x250.jpg\" alt=\"L'honorable [prénom et nom de famille]\">\n\t\t\t\t</div>\n\t\t\t\t<div class=\"col-md-7 col-xs-12\">\n\t\t\t\t\t<p>[Titre officiel du ministre]</p>\n\t\t\t\t\t<ul>\n\t\t\t\t\t\t<li><a href=\"\">Lettre de mandat [facultatif]</a></li>\n\t\t\t\t\t</ul>\n\t\t\t\t</div>\n\t\t\t</div>\n\t\t</div>\n\t</div>\n</section>"
+					}
+				]
+			}
+		}
+	],
+	"iteration": [
+		{
+			"@id": "_:iteration_ministers_1",
+			"name": "Ministers - Iteration 1",
+			"date": "2024-03",
+			"detectableBy": "section.ministers",
```

Comment:
May be we can simplify like ".gc-minister" like our CSS selector, unless that create some conflict or false positive that you are aware of.